spec.md

```diff
@@ -1104,6 +1104,17 @@ module Example:
   input intProp : Integer ; an input port of Integer property type
 ```
 
+### List Property TYpes
+
+List property types represent zero or more property values of the same type.
+List property types are parameterized over the element type.
+
+```firrtl
+module Example:
+  input list : List<Integer> ; an input port of List<Integer> property type
+  input listOfList : List<List<Integer>> ; an input port of List<Integer> property type
```

Comment:
Not sure exactly how you want to phrase it, but I think this was a copy-paste from the line above:

```suggestion
  input listOfList : List<List<Integer>> ; an input port of List<List<Integer>> property type
```